station-worldgen-api-v0/src/main/java/net/modificationstation/stationapi/impl/worldgen/WorldgenListener.java

```diff
@@ -0,0 +1,31 @@
+package net.modificationstation.stationapi.impl.worldgen;
+
+import net.mine_diver.unsafeevents.listener.EventListener;
+import net.mine_diver.unsafeevents.listener.ListenerPriority;
+import net.modificationstation.stationapi.api.StationAPI;
+import net.modificationstation.stationapi.api.event.level.biome.BiomeRegisterEvent;
+import net.modificationstation.stationapi.api.worldgen.BiomeAPI;
+import net.modificationstation.stationapi.api.worldgen.biome.BiomeProviderRegisterEvent;
+
+public class WorldgenListener {
+	private boolean initiated;
+	
+	@EventListener(priority = ListenerPriority.LOWEST)
+	public void afterInit(BiomeRegisterEvent event) {
+		if (initiated) return;
+		StationAPI.EVENT_BUS.post(new BiomeProviderRegisterEvent());
+		initiated = true;
+	}
+	
+	@EventListener
```

Comment:
Not actually using `phase = StationAPI.INTERNAL_PHASE` here.

Reply:
Fixed